python/tank/descriptor/io_descriptor/appstore.py

```diff
@@ -23,6 +23,7 @@
 
 from ...util.zip import unzip_file
 from ...util import filesystem, shotgun
+from ...util.errors import UnresolvableCoreConfigurationError
```

Comment:
I think this should be from `...util import UnresolvableCoreConfigurationError` - see other comment!